src/Calypso-Ring/RGMethod.extension.st

```diff
@@ -90,7 +90,7 @@ RGMethod >> readsSlot: aSlot [
 	| nodes |
 	nodes := self ast allChildren.
 	nodes := nodes select: #isVariable.
-	nodes := nodes select: #isInstance.
+	nodes := nodes select: #isInstanceVariable.
```

Comment:
Please use select: [:aNode | aNode isInstanceVariable]

Reply:
I will do that in the next PR